doc/source/installation.rst

```diff
@@ -32,6 +32,25 @@ Following guides describe the installation process for using the official Percon
    installation/apt_repo 
    installation/yum_repo 
 
+.. important::
+
+   When you install |Percona Server| from repositories, the installer creates a
+   backup of the :file:`/etc/mysql.cnf` file. Then, :file:`/etc/mysql.cnf`
+   becomes a soft link that ultimately (via :file:`/etc/alternatives/mysql.cnf`)
+   points to :file:`/etc/percona-server.cnf` on `RPM` based systems and to
+   :file:`/etc/mysql/percona-server.cnf` on `DEB` based systems.
+
```

Comment:
AFAIK this is different on DEB and RPM based repos, for example there is no `/etc/mysql.cnf` on debian based systems:

`root@stretch:~# ls -lah /etc/mysql.cnf
ls: cannot access '/etc/mysql.cnf': No such file or directory`

Reply:
yes for rpms and debs the message is different and paths are different